### PR TITLE
Added DEPENDENCIES.md

### PR DESCRIPTION
**Pull request by pramit-d**
https://github.com/Be-Secure/oah-bes-vm/issues/26

### CODE REVIEW

File: DEPENDENCIES.md
Comment (anilsingla) [inline DEPENDENCIES.md:1]: Can we list the packages required for this to work as well

Comment (pramit-d) [reply DEPENDENCIES.md:1]: @anilsingla As mentioned in the discussion notes, I have added all the required details in this file. Required packages/tools are Vagrant, Vagrant image/box, Ansible, Oracle VirtualBox, Ansible Roles, etc. If anything else is required please let me know.

Comment (anilsingla) [reply DEPENDENCIES.md:1]: @pramit-d We can add the required tools or packages such as Vgarant, Ansible, Linux, Python or anything which is needed to install vagrant or ansible or Oha BesVM

Comment (pramit-d) [reply DEPENDENCIES.md:1]: @anilsingla updated.

**Review by anilsingla — APPROVED** [review]

**Pull request merged**